utils/docker/dev/Dockerfile

```diff
@@ -66,7 +72,7 @@ RUN git clone https://github.com/nanopb/nanopb.git $NANOPB_PATH \
  && make
 
 # Go tools we need
-RUN go get golang.org/x/tools/cmd/goimports 
+RUN go get golang.org/x/tools/cmd/goimports
```

Comment:
Should be `go install` to pick up the requirements from module config.

Reply:
This line seems to be not needed at all. Let's remove it in a separate PR.

Reply:
some discussion on the side happened. I retract this change request.

I'm wrong here because we're not in the fpc path when this is executed. This could be resolved by moving it below WORKDIR directive.
Additionally, as goimports is part of the make process, it will automatically be created during that part of the build. This can be resolved by totally removing the RUN here.